Drop-In/src/main/java/com/braintreepayments/api/DropInClient.java

```diff
@@ -327,11 +334,18 @@ void getSupportedPaymentMethods(final FragmentActivity activity, final GetSuppor
     void getSupportedCardTypes(final GetSupportedCardTypesCallback callback) {
         braintreeClient.getConfiguration((configuration, error) -> {
```

Comment:
Moving `PaymentMethod` into core would allow us to create a `BraintreeClient#getSupportedCardTypes` method that removes DropIn's responsibility of parsing configuration for payment methods.

Reply:
Is there any equivalent currently in core for either Android or iOS?  JS does have one in core, and I agree it would make more sense to have it there, so it's both accessible to use in Drop-in, but can serve to allow merchants more control over more custom integrations.

Reply:
Look at this on iOS, turns out we currently aren't exposing `supportedCardTypes` at all. ([This](https://github.com/braintree/braintree-ios-drop-in/blob/febbaa5afb38853ff9a5bf35b0a444518777a01e/Sources/BraintreeDropIn/Helpers/BTConfiguration%2BDropIn.h) is not public)

Reply:
> Is there any equivalent currently in core for either Android or iOS?

@Epreuve we don't at the moment. A lot of the BT configuration accessors are plain `String` types. In this case, it feels like additional parsing logic in core could lessen the burden on all core-consuming modules.

> Agreed we should align on this capability across platforms.

@scannillo yeah alignment would be awesome. We will have to do a major version bump in the near future because of Cardinal. Should be a good opportunity to align the two.